### DISCUSSION TITLE
[Feature] Avg. for Top trips

### DISCUSSION THREAD

**Discussion by ogamingSCV** [teslamate-org/teslamate#3995] [Ideas]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

The data points for starting and end location is already recorded, so a list of top trips could be created, with the same start and end point. 

With all the additional data it is also possible to compare these trips to each other. Interesting for this would be for example, avg. time for trip, avg. used kWh or top start times (morning or evening). 

Furthermore a comparison via a map would show different route that have been taken. 

I imagine a "layered" map with all trips for this start/end location, with a timeline of the trip time that show the location on each path relative to the whole trip. 

### Relevant log output

```shell
-
```

### Type of installation

Docker

### Version

1.29.2

**Comment by cwanja** [comment]
Converting to discussion under **Ideas**. That is where we track feature requests. Issues are for current features that are classified as bugs. Thanks for the suggestion! We always encourage individuals to create a pull request as well.

**Comment by cwanja** [comment]
@jheredianet @DrMichael dashboard ideas.

**Comment by jheredianet** [comment]
@ogamingSCV using the existing drive details dashboard, could you make a screenshot of the info you'd like to see or is missing?
I'm not sure what you are asking for since combining other dashboards I'd say that we get what you want. 
In the other hand let's have in mind that Grafana is just like a plotter, it's not a programming language where we can merge use cases. That's why we use different dashboards depending on what you want to see instead of having all information in one graph.
